### PR TITLE
Unpost

### PR DESCRIPTION
**Pull request by XanderStrike**

### CODE REVIEW

**Comment by scuendet** [comment]
:+1:

File: lib/rentlinx/client.rb [hunk 1]
```diff
@@ -91,7 +110,8 @@ def response_handler(response)
       when 204
         nil # don't attempt to JSON parse emptystring
       when 400
-        raise Rentlinx::BadRequest, response
+        body = JSON.parse(response.body)
```
Comment (bboe) [inline lib/rentlinx/client.rb:113]: Just wanted to comment here to say "see my comment on the old commit".

Comment (XanderStrike) [inline lib/rentlinx/client.rb:113]: I'll address that in the ErrorsImprovements branch that Seb and I worked on.

Comment (d-ray) [inline lib/rentlinx/client.rb:113]: If you reference other comments, please provide links to them, particularly if the comments are on another PR.  Thanks :)

Comment (bboe) [inline lib/rentlinx/client.rb:113]: > If you reference other comments, please provide links to them, particularly if the comments are on another PR. Thanks :)

Normally I would, unfortunately I am less capable of finding the old commit hash then you are. I don't get email notifications for my own comments :(

Edit: Found it through travis: https://github.com/appfolio/rentlinx_client/commit/916831f0a410a5326f0e2d3d8842191ec3ac4f80#commitcomment-11892713

**Pull request merged**